packages/kbn-es-query/src/kuery/functions/__tests__/not.js

```diff
@@ -56,6 +62,15 @@ describe('kuery functions', function () {
         expect(result.bool).to.only.have.keys('must_not');
         expect(result.bool.must_not).to.eql(ast.toElasticsearchQuery(childNode, indexPattern));
       });
+
+      it('should accept a config parameter in a subquery in an ES bool query\'s must_not clause', function () {
+        const config = { dateFormatTZ: 'America/Phoenix' };
+        const object = { method: ast.toElasticsearchQuery };
+        const spy = sinon.spy(object, 'method');
```

Comment:
Just for future reference, after creating a sandbox you should do `sandbox.spy` to create a spy instead of `sinon.spy`.